gpt_index/readers/string_iterable.py

```diff
@@ -0,0 +1,39 @@
+from abc import abstractmethod
+from typing import Any, Iterable, List
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class StringIterableReader(BaseReader):
+    """String Iterable Reader
+
+    Gets a list of documents, given an iterable (e.g. list) of strings
```

Comment:
ubernit: add period here and above.

Reply:
thanks! https://github.com/jerryjliu/gpt_index/pull/173/commits/8dfb46c609222da687a52507e5aa0e069981899e and  https://github.com/jerryjliu/gpt_index/pull/173/commits/27a5a2e8babf6cd0f9035e1a6be57e9b4a67919a